### PR TITLE
Use 'Share' icon for `PostSharingSettings` button

### PR DESCRIPTION
**Pull request by oetherington**
Screenshot 2023-10-09 at 13 34 45

Screenshot 2023-10-09 at 13 34 04

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1205679133791410) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/editor/PostSharingSettings.tsx [hunk 1]
```diff
@@ -130,15 +143,20 @@ const PostSharingSettings = ({document, formType, value, path, label, classes}:
...
   }, [openDialog, closeDialog, formType, document, updateCurrentValues, initialSharingSettings, flash, submitForm]);
-  
-  if (!userCanUseSharing(currentUser))
-    return <LWTooltip title="You need at least 1 karma or to be approved by a mod to share">
-      <PersonAddIcon className={classes.disabledIcon}/>
+
+  const {LWTooltip} = Components;
+  if (!userCanUseSharing(currentUser)) {
+    return (
+      <LWTooltip title="You need at least 1 karma or to be approved by a mod to share">
```
Comment (agnesstenlund) [inline packages/lesswrong/components/editor/PostSharingSettings.tsx:150]:
```suggestion
      <LWTooltip title="You need at least 1 karma or to be approved by a moderator to share this post">
```

File: packages/lesswrong/components/editor/PostSharingSettings.tsx [hunk 2]
```diff
@@ -130,15 +143,20 @@ const PostSharingSettings = ({document, formType, value, path, label, classes}:
...
+        <PostSharingIcon className={classes.disabledIcon}/>
+      </LWTooltip>
+    )
+  };
+  return (
+    <LWTooltip title="Share this document">
```
Comment (agnesstenlund) [inline packages/lesswrong/components/editor/PostSharingSettings.tsx:156]:
```suggestion
    <LWTooltip title="Share this post">
```

**Review by agnesstenlund — COMMENTED** [review]
sneaking these changes in too

**Comment by agnesstenlund** [comment]
Thank you!!

**Review by jpaddison3 — APPROVED** [review]

**Pull request merged**